### ISSUE TITLE
How to get URL of image only and not download?

### ISSUE THREAD

**Issue by jeraldpunx** [hardikvasa/google-images-download#167]
I want to get the URL only and don't download and put it on JSON just for exp

```
{
    "Records": [
        {
            "keywords": "Lancome Nutrix Nourishing and Repairing Treatment Rich Cream 75ml",
            "limit": 1,
            "safe_search": true,
        },
        {
            "keywords": "Tom Ford Black Orchid Eau de Parfum 50ml Spray",
            "limit": 1,
            "safe_search": true,
        }
    ]
}
```

will return me 

```
{
    "Records": [
        {
            "keywords": "Lancome Nutrix Nourishing and Repairing Treatment Rich Cream 75ml",
            "limit": 1,
            "safe_search": true,
            "image_url": "https://www.lancome.co.uk/dw/image/v2/AAQP_PRD/on/demandware.static/-/Sites-lac-master-catalog/default/dwf7c52d54/images/PACKSHOTS/SKINCARE/NUTRIX/002012_Nutrix/3147755503152_Nutrix_125ml.jpg?sw=750&sh=750&sm=fit&sfrm=png"
        },
        {
            "keywords": "Tom Ford Black Orchid Eau de Parfum 50ml Spray",
            "limit": 1,
            "safe_search": true,
             "image_url":"https://images-na.ssl-images-amazon.com/images/I/61NgsdttMrL._SL1000_.jpg"
        }
    ]
}

```

if not, can I get the JSON log with file name?

**Comment by hardikvasa** [comment]
@jeraldpunx this is now available in #169 . Closing this issue.

**Comment by jeraldpunx** [comment]
I mean extracting all url into json, not print in console. it shows me in console only and on my download folder, it creates a folder name of a keyword.
